pallets/stateful-storage/src/tests/other_tests.rs

```diff
@@ -0,0 +1,211 @@
+use crate::{
+	stateful_child_tree::StatefulChildTree,
+	test_common::{constants::*, test_constants::*},
+	tests::mock::*,
+	types::*,
+	Config, Error,
+};
+use codec::Encode;
+use common_primitives::utils::wrap_binary_data;
+use frame_support::{assert_err, assert_ok, BoundedVec};
+#[allow(unused_imports)]
+use pretty_assertions::{assert_eq, assert_ne, assert_str_eq};
+use sp_core::Pair;
+use sp_runtime::MultiSignature;
+
+#[test]
+fn is_empty_false_for_non_empty_page() {
+	let page: ItemizedPage<Test> =
+		create_itemized_page_from::<Test>(None, &[generate_payload_bytes(None)]);
+
+	assert_eq!(page.is_empty(), false);
+}
+
+#[test]
+fn is_empty_true_for_empty_page() {
+	let page: ItemizedPage<Test> = create_itemized_page_from::<Test>(None, &[]);
+
+	assert_eq!(page.is_empty(), true);
+}
+
+#[test]
+fn apply_actions_on_signature_schema_fails() {
```

Comment:
This should be in apply_item_actions file and test name might need to change to be compatible with the convention

Reply:
I will change it to `apply_item_actions_on_signature_schema_fails()` and move it.